rstudio_cheatsheet.Rmd

```diff
@@ -0,0 +1,8 @@
+# RStudio Cheatsheet
+
+Mendel Branover
+
+This document serves as a cheatsheet and guide for using the Studio IDE. It includes brief explanations of useful features in RStudio. 
+The document can be used as an introduction for new Rstudio users. It can also be used as a quick start guide for any RStudio user wanting a brief breakdown and some shortcuts.
+
+The URL for the document is: https://github.com/mbran1/GR5702-EDAV
```

Comment:
@jtr13 I wanted to confirm if submitting a link like this would be fine or should it be in ![](<link>) format

Reply:
Yes, that's fine. It will work.